cmd/common/profile/cpu.go

```diff
@@ -15,14 +15,28 @@
 package profile
 
 import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
 	"github.com/spf13/cobra"
+
+	"github.com/inspektor-gadget/inspektor-gadget/cmd/common/utils"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/profile/cpu/types"
```

Comment:
nit: we're trying to name then `<gadget>Types`, then it should be `cpuTypes`.

Reply:
Ok. I was under the impression that we should only name then when package name clashes. Changed